bus-mapping/src/circuit_input_builder/transaction.rs

```diff
@@ -197,6 +197,10 @@ pub struct Transaction {
     pub input: Vec<u8>,
     /// Signature
     pub signature: Signature,
+    /// Invalid tx
+    pub invalid_tx: Option<bool>,
```

Comment:
Is None type usable somewhere? why is this an option but the other one in another tx type is bool?

Reply:
Removed Option.